# Bioinformatics-internship 

## ("Salmonella" OR "S.") AND ("typhi" OR "typhoid") AND ("antimicrobial resistance" OR "antibiotic resistance" OR "AMR")

## Data collection related to "Salmonella enterica" AND "antimicrobial resistance"

A general workflow for AMR monitoring using bioinformatics tools:

Step 1: Data Collection

- Collect bacterial isolates from various sources (e.g., clinical, environmental, animal)

- Extract genomic DNA from the isolates

Step 2: Genome Assembly

- Use genome assembly tools (e.g., SPAdes, Velvet) to reconstruct the bacterial genome
from sequencing data

- Evaluate assembly quality using metrics (e.g., N50, contig length)

Step 3: Genome Annotation

- Use genome annotation tools (e.g., Prokka, NCBI Genome Annotation) to predict protein-
coding genes and functional annotations

- Integrate annotations with external databases (e.g., CARD, ARG-ANNOT)
https://db.cngb.org/search/sample/?q=PRJNA289090


Step 6: MLST and cgMLST Analysis

- Use MLST and cgMLST analysis tools (e.g., MLST, cgMLST) to determine the sequence type
(ST) and core genome multilocus sequence type (cgMLST) of each isolate

- Evaluate the genetic diversity and population structure of the bacterial isolates

## Data disturbution 

https://www.ncbi.nlm.nih.gov/pathogens/isolates/#%22Salmonella%20enterica%22%20AND%20%22antimicrobial%20resistance%22

https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA998920


https://www.ncbi.nlm.nih.gov/pathogens/isolates/#(%22Salmonella%22%20OR%20%22S.%22)%20AND%20(%22typhi%22%20OR%20%22typhoid%22)%20AND%20(%22antimicrobial%20resistance%22%20OR%20%22antibiotic%20resistance%22%20OR%20%22AMR%22)

# Refrences
https://elifesciences.org/articles/85867/figures#content

https://data.mendeley.com/research-data/?type=DATASET&search=%22Salmonella%20typhi%22%20AND%20%22antimicrobial%20resistance%22

https://www.typhoidgenomics.org/
