# Bioinformatics-internship 

# "Salmonella typhi" AND "antimicrobial resistance"

## Data collection related to "Salmonella enterica" AND "antimicrobial resistance"

A general workflow for AMR monitoring using bioinformatics tools:

Step 1: Data Collection

- Collect bacterial isolates from various sources (e.g., clinical, environmental, animal)

- Extract genomic DNA from the isolates

Step 2: Genome Assembly

- Use genome assembly tools (e.g., SPAdes, Velvet) to reconstruct the bacterial genome
from sequencing data

- Evaluate assembly quality using metrics (e.g., N50, contig length)

Step 3: Genome Annotation

- Use genome annotation tools (e.g., Prokka, NCBI Genome Annotation) to predict protein-
coding genes and functional annotations

- Integrate annotations with external databases (e.g., CARD, ARG-ANNOT)
https://db.cngb.org/search/sample/?q=PRJNA289090

## Data disturbution 

https://www.ncbi.nlm.nih.gov/pathogens/isolates/#%22Salmonella%20enterica%22%20AND%20%22antimicrobial%20resistance%22

https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA998920

# Refrences
https://elifesciences.org/articles/85867/figures#content

https://data.mendeley.com/research-data/?type=DATASET&search=%22Salmonella%20typhi%22%20AND%20%22antimicrobial%20resistance%22

https://www.typhoidgenomics.org/
